vm_server/accept/server.py

```diff
@@ -18,28 +19,72 @@
 
 sem = threading.Semaphore()
 
-def remove_execute_dir():
-  """Deletes the execute directory if it exists"""
+def remove_execute_dir(task_response):
+  """Deletes the execute directory if it exists
+  
+  Args:
+    task_response: an object of TaskResponse() that will be sent back
+  """
+  logging.debug("Removing execute directory")
   dirpath = Path("..\\execute")
-  if dirpath.exists() and dirpath.is_dir(): # delete leftover files
-    shutil.rmtree(dirpath)
+  try: 
+    if dirpath.exists() and dirpath.is_dir(): # delete leftover files
+      shutil.rmtree(dirpath)
+  except Exception as exception:  #catch errors if any
+    print(exception)
```

Comment:
can remove print since you are logging, also same for all other print statements

Reply:
REmoved print statement